teaclave_attestation/src/key.rs

```diff
@@ -0,0 +1,167 @@
+use anyhow::Result;
+use sgx_tcrypto::SgxEccHandle;
+use sgx_types::{sgx_ec256_private_t, sgx_ec256_public_t};
+use std::prelude::v1::*;
+
+pub const CERT_VALID_DAYS: i64 = 90i64;
+
+pub struct Secp256k1KeyPair {
+    pub prv_k: sgx_ec256_private_t,
```

Comment:
It seems that the prv_k field is not directly used by other modules, is it OK to remove the pub modifier?